lib/llvm/Support/Debug.cpp

```diff
@@ -78,7 +78,10 @@ void setCurrentDebugTypes(const char **Types, unsigned Count) {
 } // namespace llvm
 
 // All Debug.h functionality is a no-op in NDEBUG mode.
-#ifndef NDEBUG
+//
+// LLBUILD-ONLY: Debug.h macros are disabled for llbuild (they inject static
+// constructors into library code).
+#if !defined(NDEBUG) && defined(LLVM_CODE_DISABLED_FOR_LLBUILD)
 
```

Comment:
`!defined(LLVM...` ?

ie. a debug build, but not for llbuild

Reply:
Oh, I see, it’s expected to be undefined for our case. 

The double negatives are strong with this one.

Reply:
Ya, its gross, but easy